cmd/juju/waitfor/query/parser_test.go

```diff
@@ -365,3 +369,32 @@ func (p *parserSuite) TestParserInfixLambda(c *gc.C) {
 		},
 	})
 }
+
+func Fuzz(f *testing.F) {
```

Comment:
Can we give this test a post fix? Like FuzLexer or something?

Reply:
Sure can